public/assets/js/game.js

```diff
@@ -0,0 +1,120 @@
+// default screen, start button to begin/reset game
```

Comment:
This code is not very DRY. We have the same calls to disable buttons or set/change health. Consider moving repeated functionality into functions.